src/main/java/com/xpandit/plugins/xrayjenkins/task/XrayImportBuilder.java

```diff
@@ -583,6 +576,112 @@ public void perform(
                 .setAction(build, listener);
     }
 
+    private void importResultsSequential(
+            Run<?, ?> build,
+            FilePath workspace,
+            TaskListener listener,
+            XrayInstance importInstance,
+            XrayImporter client,
+            EnvVars env,
+            List<UploadResult> uploadResults,
+            List<FilePath> files
+    ) throws IOException, InterruptedException {
+        String key = null;
+        if (CollectionUtils.isEmpty(files)) {
```

Comment:
can you invert the logic in this _if_ condition?
```java
if (CollectionUtils.isNotEmpty(files)) {
    for (FilePath fp : files) {
        ....
    } 
}
```

Reply:
I like this way, to me it's more clear, however, I can change. What do you think?